libs/utils/utils.js

```diff
@@ -965,3 +965,5 @@ export function loadLana(options = {}) {
   window.addEventListener('error', lanaError);
   window.addEventListener('unhandledrejection', lanaError);
 }
+
+window.utilsloaded = performance.now();
```

Comment:
Are we using this anywhere else? Name feels a bit generic to me, maybe we can scope it to something related to 'milo'

Reply:
ugh thanks for seeing that.  Leftover debug code.